utils/sambanova_endpoint.py

```diff
@@ -1070,24 +1076,26 @@ def _handle_nlp_predict_stream(
             }
 
             if chunk['status_code'] == 200 and chunk.get('error'):
-                chunk['result'] = {'responses': [{'stream_token': ''}]}
+                raise RuntimeError(
+                    f"Sambanova /complete call failed with status code " f"{chunk['status_code']}." f"{chunk}."
+                )
+                
             if chunk['status_code'] != 200:
```

Comment:
We can evaluate as alternative to have an if/else here. So if  !=200 else ==200. Then in the latter, if chun.get('error') else "do the rest of stuff"

Reply:
right updated, also added  handling of non expected errors